src/pages/Dashboard.tsx

```diff
@@ -64,8 +68,28 @@ class Dashboard extends Component<Props, DashboardState> {
       path: "configuration"
     });
     this.setState({ applicationConfig });
+    await this.fetchScheduledInfo();
   }
 
+  private fetchScheduledInfo = async () => {
+    try {
+      const response = await fetch("./scheduled-info.json", {
+        cache: "no-cache"
+      });
+      if (!response.ok) {
+        throw new Error("Error while loading scheduled info data");
+      }
+      const data = await response.json();
+
+      ScheduledInfoList.decode(data).fold(
+        err => console.error("Error while decoding scheduled info", err),
+        scheduledInfoBanners => this.setState({ scheduledInfoBanners })
```

Comment:
Why are you renaming a the sheduledinfo array to "banner"?

Reply:
No particular reasons other than to give it a more "UI" value 🙂
Anyway I renamed it to `scheduledInfoList` 👉 5859343